Lib/test/test_platform.py

```diff
@@ -1,11 +1,16 @@
+import io
+import itertools
 import os
 import copy
+import contextlib
 import pickle
```

Comment:
Let's keep some sort of alphabetical order:
```suggestion
import os
import copy
import contextlib
import io
import itertools
import pickle
```